docs/sources/structure/_index.md

```diff
@@ -1,33 +1,40 @@
 ---
-title: Documentation structure
-description: How to organize concepts and tasks in the repository.
-weight: 300
-menuTitle: Structure
 aliases:
   - /docs/writers-toolkit/writing-guide/documentation-structure/
   - /docs/writers-toolkit/structure
+description: How to organize concepts and tasks in the repository.
 keywords:
   - information architecture
   - structure
+menuTitle: Structure
+title: Documentation structure
+weight: 300
 ---
 
 # Documentation structure
 
-The Grafana Labs Docs team makes intentional decisions about how to organize and structure product documentation. The topic levels discussed on this page reflect common user goals. For example, a first-time Grafana user needs to understand the basic concepts before installing the product.
+The Grafana Labs Docs squad makes intentional decisions about how to organize and structure product documentation.
+The topic levels discussed on this page reflect common user goals.
+For example, a first-time Grafana user needs to understand the basic concepts before installing the product.
 
-Before you begin contributing to documentation, it is important to understand the structure of the content.
+Before you begin contributing to documentation, it's important to understand the structure of the content.
 
 ## User goals and documentation structure
 
-When writing documentation, focus on what your user's goals are. For more information, refer to [Focus on user goals]({{< relref "../write/style-guide/style-conventions" >}}).
+When writing documentation, focus on what your user's goals are.
+For more information, refer to [Focus on user goals](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#focus-on-user-goals).
```

Comment:
Could we combine these into one?

```suggestion
When writing documentation, [focus on what your user's goals](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#focus-on-user-goals) are.
```

Reply:
I think that paragraph makes more sense in this page than where it is currently. @chri2547 do you have objections to me moving that paragraph here?

Reply:
@jdbaldry , I'm unclear on what you mean. Which paragraph do you want to move here? And where is it currently?

Reply:
Ah, I appear to have misunderstood Clayton's comment, apologies to you both. 

I was thinking that the proposal was to move the [Focus on user goals](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#focus-on-user-goals) section to this page but in fact it's just to combine the two sentences.

Reply:
Gotcha, all good. Approved!